recipes/shutdown.rb

```diff
@@ -1,6 +1,7 @@
 # Ensure deletes the node and client upon shutdown
 #
-cookbook_file '/etc/init.d/chef_lifecycle' do
+template '/etc/init.d/chef_lifecycle' do
+  source 'shutdown/chef_lifecycle.erb'
   mode '0750'
```

Comment:
Does this need a `variables` parameter to pass `node['cfn']['shutdown']['node_name']` to the template as `node_name` ?